daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -532,11 +531,11 @@ private[archive] class DecodeV1(minor: LanguageMinorVersion) extends Decode.OfPa
           CPCons(name(cons.getVarHead), name(cons.getVarTail))
 
         case PLF.CaseAlt.SumCase.OPTIONAL_NONE =>
-          assertSince("1", "CaseAlt.OptionalNone")
+          assertSince(optionalVersion, "CaseAlt.OptionalNone")
```

Comment:
Why have a "named" alias specifically for version `1.1` but not some others? The name `optionalVersion` also reads more like that the version is optional. Do you want to make the move to "true" feature flags later, or stick with language aliases in `Features`?

Reply:
I created alias for all version.